esmvaltool/recipes/recipe_zmnam.yml

```diff
@@ -34,20 +34,17 @@ preprocessors:
     regrid:
       target_grid: 3x3
       scheme: area_weighted
-    extract_region:
-      start_longitude: 0.
-      end_longitude: 360.
-      start_latitude: 20.
-      end_latitude: 90.
-
 
 diagnostics:
   zmnam:
-    description: Zonal mean Northern Annular Mode
+    description: Zonal mean annular mode
     variables:
       zg:
         preprocessor: preproc
     scripts:
       main:
         script: zmnam/zmnam.py
-        title: "Zonal mean NAM diagnostics"
+        title: "Zonal mean AM diagnostics"
+        hemisphere: "NH" # NH or SH
+        write_plots: true
+        write_netcdf: true # currently not used
```

Comment:
Thanks for the comments! Yes that should be possible with the current workflow. Let me check...

Reply:
By the way, is there a place (e.g. example recipe or documentation) where one can read all the 'standard' settings such as `write_netcdf`?

Reply:
The closest to that would be the [description of the esmvalcore-diagnostic interface](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/interfaces.html#information-provided-by-esmvalcore-to-the-diagnostic-script), but it's not entirely complete.

These settings come from [config-user.yml](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/configure.html#user-configuration-file) after some modification and are passed on to the diagnostic script. Because many diagnostic scripts did not implement `write_plots` and `write_netcdf` and they were not very useful settings anyway (the savings in time is very minimal in most cases), we decided to remove them, see https://github.com/ESMValGroup/ESMValCore/issues/93.

The only remaining settings are `output_file_type` for defining the plot file type (we also wanted to remove that, but it proved too much work) and `log_level`, which is automatically handled by the [`run_diagnostic`](https://docs.esmvaltool.org/en/latest/api/esmvaltool.diag_scripts.shared.html#esmvaltool.diag_scripts.shared.run_diagnostic) function for setting up the logger.

Reply:
Thanks, that's useful to know!